plugins/woocommerce/includes/admin/helper/class-wc-helper-plugin.php

```diff
@@ -16,15 +16,25 @@
  * Contains the logic to manage the Woo Marketplace plugin.
  */
 class WC_Helper_Plugin {
-	const WOO_MARKETPLACE_PLUGIN_MAIN_FILE = 'woo-marketplace/woo-marketplace.php';
+	const WOO_UPDATE_MANAGER_PLUGIN_MAIN_FILE = 'woo-marketplace/woo-marketplace.php';
+	const WOO_UPDATE_MANAGER_DOWNLOAD_URL     = 'https://woo.com/products/woo-update-manager/';
```

Comment:
Would it be possible to use relative paths to the Woo.com instead of the absolute full URL? This will reduce the number of replacements needed during domain migration.

Reply:
There are many occurrences of `https://woo.com/` urls in admin/helper code. I think this is not the PR to fix that. Maybe if we had to do a domain migration again, we can make sure there urls are handled in a better way when replacing the domain.